uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlementTest.php

```diff
@@ -688,6 +688,43 @@ private function getEntitlementCalculationMock(
     return $calculation;
   }
 
+	public function testBalanceShouldIncludeOpenLeaveRequests() {
```

Comment:
This method name is wrong. Only the future balance should include open leave requests. I think a better name would be something like `testFutureBalanceShouldIncludeApprovedAndOpenLeaveRequests()`